app/assets/javascripts/discourse/app/controllers/bookmark.js

```diff
@@ -302,8 +297,22 @@ export default Controller.extend(ModalFunctionality, {
     return this.nextMonth().format(I18n.t("dates.long_no_year"));
   },
 
-  get postLocalDateFormatted() {
-    return this.postLocalDate().format(I18n.t("dates.long_no_year"));
+  loadLocalDates() {
+    let postEl = document.querySelector(
```

Comment:
This is looking excellent but I see we still have `document` usage in a controller here.

Reply:
@eviltrout sorry to butt in but I want to learn this too. Should accessing the document/elements on the page only ever happen in a component? If so that means I originally should have put this bookmark code into a component within the modal? Just asking because I am also doing a refactor of this bookmark stuff in another PR (and for future Ember stuff anyway)